src/bin/vip-import-check.js

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env node
+// @flow
+
+/**
+ * External dependencies
+ */
+const readline = require( 'readline' );
+const fs = require( 'fs' );
+import chalk from 'chalk';
+const log = require( 'single-line-log' ).stdout;
+
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+
+let problemsFound = 0;
+let lineNum = 1;
+
+const errorCheckFormatter = ( check ) => {
+	if ( check.results.length > 0 ) {
+		problemsFound += 1;
+		console.error( chalk.red( 'Error:' ), `${ check.message } on line(s) ${ check.results.join( ',' ) }.` );
+		console.error( chalk.yellow( 'Recommendation:' ), `${ check.recommendation }` );
+	} else {
+		console.log( `✅ ${ check.message } was found ${ check.results.length } times.` );
+	}
+};
+
+const requiredCheckFormatter = ( check, type ) => {
+	if ( check.results.length > 0 ) {
+		console.log( `✅ ${ check.message } was found ${ check.results.length } times.` );
+		if ( type === 'createTable' ) {
+			checkTablePrefixes( check.results );
+		}
+	} else {
+		problemsFound += 1;
+		console.error( chalk.red( 'Error:' ), `${ check.message } was not found.` );
+		console.error( chalk.yellow( 'Recommendation:' ), `${ check.recommendation }` );
+	}
+};
+
+const infoCheckFormatter = ( check ) => {
```

Comment:
Interesting-ish find here (eslint/wp-prettier):
 
* on eslint --fix I see the parens go away and some spacings change
* on save (and auto wp-prettier) I see them come back, and spacings change back

Reply:
It's probably a conflict between eslint and prettier rules? I believe we have an eslint rule that doesn't require parens for single param arrow functions.